serverlessworkflow/modules/ROOT/pages/cloud/operator/known-issues.adoc

```diff
@@ -60,7 +56,5 @@ The following issues is currently being prioritized.
 === Knative Integration
 
 - link:https://issues.redhat.com/browse/KOGITO-8648[Implement the Knative Addressable interface in dev profile]
-- link:https://issues.redhat.com/browse/KOGITO-8409[Add support to Cloud Events to Knative custom function]
 - link:https://issues.redhat.com/browse/KOGITO-8410[Add support to GET method to Knative custom function]
-- link:https://issues.redhat.com/browse/KOGITO-8766[Adjust the Knative Function definition to use the same interface as defined by the extension]
 - link:https://issues.redhat.com/browse/KOGITO-8646[Review the need of the knative/kubernetes addons inside the dev profile builder image]
```

Comment:
These are only implemented in the runtimes core. We don't have any support for these features in the operator.